RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -596,6 +596,15 @@ class FeedbackItem extends React.Component<IFeedbackItemProps, IFeedbackItemStat
     />);
   };
 
+  private formatTimer = (timeInSeconds: number) => {
+    // Handle the timer display
+    // Doesn't handle formatting hours since that may be excessive
+    const timerMinutes = Math.floor(timeInSeconds / 60);
+    const timerSeconds = timeInSeconds % 60;
+    const leadingZeroSeconds = timerSeconds < 10;
```

Comment:
nice touch with displaying the extra '0' :)

Reply:
the extra zero is awesome! I'd just suggest renaming `leadingZeroSeconds` to something like `showLeadingSecondsZero` to follow the naming conventions in the rest of this file

Reply:
Thank you! Oh, excellent point, I'll adjust the name!